docs/doc/en/Application/Creation_d_un_module.md

```diff
@@ -56,7 +56,7 @@ A module therefore has :
 - a name (_name_) to give the module an identity key. This name must be unique among the modules used in the application, to avoid confusion.
 - a title (_title_) to be displayed to the user on the drawer when no app is selected.
 - a subtitle (_subtitle_) to be displayed to the user under the module button in the drawer to aid comprehension. The display of these subtitles can be configured in the _Application_ component using the _showModulesSubtitle._ attribute.
-- an icon name (_icon_) to be displayed on the module button in the drawer. Please note that this icon must belong to the FontAwesome 5 database of [react-native-vector-icons](https://oblador.github.io/react-native-vector-icons/) package.
+- an icon name (_icon_) to be displayed on the module button in the drawer. Please note that this icon must belong to the Bootstrap database (https://icons.getbootstrap.com/) package.
```

Comment:
Don't remove [ ] before ( ) in md
Plus, I think it's better to write: "this icon must belong to the [Bootstrap database] (...)." (without space between [ ] and ( ))